### ISSUE TITLE
install fails at "write ini files values"

### ISSUE THREAD

**Issue by JimFlasch** [snowie2000/mactype#305]
Suddenly (as of today), no version of MacType will install on my system. All will fail at "write ini files values,"
On top of that, the new version, which I tried first, failed to roll back the previous version, levaing me with crappy font rendering in certain programs! Why is that?

**Comment by sammilucia** [comment]
Hi @JimFlasch ... sorry for no reply. Did you ever work this out?

**Comment by sammilucia** [comment]
yup! got out of hand for a while, but we're getting on top of things now. thanks for your response @JimFlasch ! 😊
